index.js

```diff
@@ -64,19 +53,21 @@ function push() {
 // Returns argument required to install helm-s3 and helm-pack plugins.
 function installPlugins() {
   const plugins = [
-    'https://github.com/hypnoglow/helm-s3.git',
-    'https://github.com/thynquest/helm-pack.git',
+    { url: 'https://github.com/hypnoglow/helm-s3.git', version: 'v0.12.0' },
+    { url: 'https://github.com/thynquest/helm-pack.git', version: 'v0.2.2' },
   ];
 
-  return Promise.all(plugins.map(plugin => exec.exec(HELM, ['plugin', 'install', plugin])));
+  return Promise.all(
+    plugins.map((plugin) =>
+      exec.exec(HELM, ['plugin', 'install', plugin.url, '--version', plugin.version])
+    )
+  );
```

Comment:
this block is the only real change in this PR. the other diffs are auto-formatting.